pkgs/development/python-modules/radicale_infcloud/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "radicale_infcloud";
+  name = "${pname}-${version}";
+  version = "972757bf4c6be8b966ee063e3741ced29ba8169f";
```

Comment:
We prefer date of commit here, since it is human readable and can be ordered by `nix-env`.

Reply:
Updated. I'm never sure about this, I've seen a bunch of conflicting recommendations for what to do for versions straight from VCS.